homework/Anastasia_Kichaeva/Hw4_Task1/src/main/java/acya/ki/service/SignComporator.java

```diff
@@ -0,0 +1,37 @@
+package acya.ki.service;
+
+import acya.ki.bean.SignComporatorBean;
+
+public class SignComporator {
+    private SignComporatorBean signComporatorBean; //объявляю объект бин в поле класса сервис
+
+    public SignComporator() { // пустой конструктор сервиса
+    }
+
+
+    public SignComporator(SignComporatorBean signComporatorBean) { //конструктор со значением сервиса
+        this.signComporatorBean = signComporatorBean;
+    }
+
+    public String compare(SignComporatorBean signComporatorBean) {  //не кладу в скобки ничего, потому что создала конструктор
+        if (signComporatorBean.getNumber() > 0) {
+            return ("Number is positive");
+        } else if (signComporatorBean.getNumber() < 0) {
+            return ("Number is negative");
+        } else {
+            return ("Number = 0");
+        }
+    }
+    public void result(SignComporatorBean signComporatorBean) {
```

Comment:
You have two same methods. Why?

Reply:
because i want see text from result method in consol. Can i add System.out.println in String compare method(under return) ?